test/autests/gold_tests/body_verification/http.yaml

```diff
@@ -0,0 +1,140 @@
+meta:
+  version: '1.0'
+sessions:
+- protocol:
+  - name: http
+    version: 1.1
+  - name: tls
+    sni: test_sni
+  - name: tcp
+  - name: ip
+    version: 4
+  transactions:
+  - all:
+      headers:
+        fields:
+        - [uuid, 1]
```

Comment:
This `all` approach was used earlier on. The client and server are updated now to just be able to use the key in the `client-request` node now. Thus, just put these `all:headers:fields:uuid` values in `client-request:headers:fields:uuid`.